.changeset/fair-buttons-punch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": minor
+---
+
+`<rh-card>`: Add Inline Promo demo + styles
```

Comment:
these changesets should be user-facing, instead of developer-facing

so, instead of listing what you did as a programmer here ("added demo and styles"), instead say what was added (:sparkles: Added inline promo variant) and put a short description of the feature and a fenced code block demonstrating it's typical usage

Reply:
Updated in [f603b8e](https://github.com/RedHat-UX/red-hat-design-system/pull/1569/commits/f603b8e3dcc56eefc05748b6da40a4b5d6a68ed1). 

AFAICT, this is the first variant to follow this convention. Double check to make sure this is the correct way to do it.

Reply:
looks good but might need updating after 75da1c74251ad775c50f8ea94fd90a1e28199d8d

Reply:
Updated changeset in [cef0ff3](https://github.com/RedHat-UX/red-hat-design-system/pull/1569/commits/cef0ff3bd8fb203f32c7f6585562bd3dad59d738) to match 1:1 to the demo.